docs/docs/self-hosting/configuration/application.md

```
@@ -0,0 +1,194 @@
+- Comma-separated IPs/CIDRs: `10.0.0.1,192.168.1.0/24` - Trust specific proxies only
+- Empty - Trust no proxies
+
+:::info
+Checks the [Troubleshooting section](#troubleshooting) for help with proxy configuration.
+:::
+
+## Logging
+
+| Variable | Description | Default |
+|----------|-------------|---------|
+| `LOG_CHANNEL` | Logging channel | `stderr` |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)